src/components/material-buttons/MaterialListButton.jsx

```diff
@@ -0,0 +1,22 @@
+import { MaterialButton } from './MaterialButton';
+
+export const whiteFontColor = {
+	color: 'white',
+};
+
+export const buttonStyle = {
+	width: '15em',
+	fontSize: '1.5rem',
+	...whiteFontColor,
+};
+
+export const MaterialListButton = ({ buttonText, onClick }) => {
+	return (
+		<MaterialButton
+			sx={buttonStyle}
+			type="button"
+			onClick={onClick}
+			buttonText={buttonText}
+		/>
```

Comment:
Same question as https://github.com/the-collab-lab/tcl-75-smart-shopping-list/pull/72/files#r1789249110